### PR TITLE
Fix broken docs links.

### LINKED ISSUE #54: Broken link in documentation
`core-api.html` should be `core-get.html`: https://github.com/oddbird/accoutrement/blob/5b674faa4d68db2413c282267ffd51ce69032ce5/sass/_core.scss#L73

### PR DESCRIPTION
**Pull request by jgerigmeyer**
Fixes #54.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -40,11 +40,12 @@ As you work...
   using [True](http://oddbird.net/true) --
   in the `test` directory,
   and make sure all tests are passing with `yarn test`
-- Add your changes to the [changelog](CHANGELOG.md)
+- Add your changes to the [changelog][]
 - Update or add inline documentation
   using [SassDoc][sassdoc] as appropriate,
   and compile the docs with `yarn docs`
 
+[changelog]: https://github.com/oddbird/accoutrement/blob/master/CHANGELOG.md
```
Comment (mirisuzanne) [inline CONTRIBUTING.md:48]: Better to link them in the docs rather than on a particular gihub branch (especially one we'll likely change soon)?

Comment (jgerigmeyer) [reply CONTRIBUTING.md:48]: Up to you.

Comment (mirisuzanne) [reply CONTRIBUTING.md:48]: let's link to docs

Comment (jgerigmeyer) [reply CONTRIBUTING.md:48]: Ok -- the only thing that seems odd about that is this is in the Contributing doc, referencing a file to edit (not just read). 🤷

Comment (mirisuzanne) [reply CONTRIBUTING.md:48]: oh, that's true. Ok, we can update the branch once we change it…

**Pull request merged**